financialaid/serializers.py

```diff
@@ -0,0 +1,62 @@
+"""
+Serializers from financialaid
+"""
+import datetime
+from django.shortcuts import get_object_or_404
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from rest_framework.fields import IntegerField, FloatField, CharField
+
+from courses.models import Program
+from dashboard.models import ProgramEnrollment
+from financialaid.api import determine_tier_program, determine_auto_approval
+from financialaid.models import FinancialAid, FinancialAidStatus
+
+
+class FinancialAidSerializer(serializers.Serializer):
```

Comment:
But `program` is on the `FinancialAid` model though...

Anyway in the interest of speeding this up feel free to leave it, but this solution throws out a lot of free validation.